treeherder/etl/push_loader.py

```diff
@@ -95,20 +96,16 @@ def fetch_push(self, url, repository):
         head_commit = commits[-1]
         push = {
             "revision": head_commit["sha"],
+            # A push can be co-authored
+            # The author's date is when the code was committed locally by the author
+            # The committer's date is the info as to when the PR is merged (committed) into master
             "push_timestamp": to_timestamp(
-                head_commit["commit"]["author"]["date"]),
+                head_commit["commit"]["committer"]["date"]
```

Comment:
You might laugh but the only real code change is this single line.
Everything else is just to enable manual ingestion and validation.